src/psse_export.jl

```diff
@@ -265,9 +265,21 @@ convert_empty_stringvec = Base.Fix1(convert_empty, Vector{String})
 # PERF could be improved by appending to the buffer rather than doing string interpolation, seems unnecessary
 _psse_quote_string(s::String) = "'$s'"
 
+# Rounds a value up to 4 decimals, to avoid large approximations in file
+_psse_round_val(val) = isa(val, String) ? val : round(val; digits = 4)
```

Comment:
do this with multiple dispatch, also maybe define a const for digits we want to round to

Reply:
Also, what's your rule for when this gets used and when it doesn't? Let's be consistent